packages/app/background-jobs-common/src/background-job-processor/processors/AbstractBackgroundJobProcessor.spec.ts

```diff
@@ -527,9 +520,6 @@ describe('AbstractBackgroundJobProcessor', () => {
           errorJson: expect.stringContaining(onFailedCall.error.message),
         },
       })
-      expect(
```

Comment:
do we no longer find this assertion valuable? how do we know childloggers work as expected?

Reply:
It is valuable, but we can't test it easily from `AbstractBackgroundJobProcessor` as `resolveExecutionLogger` is no longer visible at this level, I was thinking of adding those checks at `BackgroundJobProcessorMonitor` but it is not there yet (those tests are my next priority on this PR), does it sound good?

Reply:
yeah, sounds good!